config/crd/kustomization.yaml

```diff
@@ -4,19 +4,22 @@
 resources:
 - bases/v1beta1/datadoghq.com_datadogagents.yaml
 - bases/v1beta1/datadoghq.com_datadogmetrics.yaml
+- bases/datadoghq.com_datadogmonitors.yaml
```

Comment:
should we have also `v1beta1` in the path too?

Reply:
looks like it, thank you